content/log/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+{{< img src="log/index/pipeline_sketch.png" alt="Pipelines sketch" >}}
+
+## Getting started with the Agent
+
+Collecting logs is **disabled** by default in the Datadog-Agent.
+In order to start to gather your logs you need to:
+
+1. [Install the latest Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+/////\\\\\\<br>
+/////\\\\\\<br>
+/////\\\\\\Add specific steps once we have the definitive process<br>
+/////\\\\\\<br>
+/////\\\\\\<br>
+
+## Enabling log collection from integrations
+
+To start collecting logs for an integration, you need to uncomment the log sections in your integration yaml file, and then set its parameters values according to your infrastructure.
+
+If you don't find a log section in your file you need to use the custom file configuration below instead. 
+
+## Custom log collection
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit those parameter:
+* `type` : (manatory) type of log input source (**tcp** / **udp** / **file**)
+* `port` / `path` : (mandatory) option depending on `type`.
+* `service` : (mandatory) name of the service owning the log
+* `source` : (mandatory) attribute that defines which integration is sending the logs. It can be a custom one if the integration does not exist
+* `sourcecategory` : (optional) Multiple value attribute. Can be used to refine the source attribtue. Example: source:mongodb, sourcecategory:db_slow_logs
+* `tags`: (optional) add tags to each logs collected.
+
+### Tail existing files
+Set `type` to **file** then specify the absolute `path` to the log file you want to tail.
+
+Example: 
+If you want to gather you nginx logs for instance stored in **/var/log/access.log** and **/var/log/error.log** you would edit your `custom-logs.yaml` file as follow
+
+{{< highlight yaml >}}
+#Log section
+logs:
+  - type: file
+    path: /var/log/access.log
+    service: nginx
+    source: nginx
+    sourcecategory: http_access
+    tags: env:prod 
+
+  - type: file
+    path: /var/log/error.log
+    service: nginx
+    source: nginx
+    sourcecategory: http_error
+{{< /highlight >}}
+
+### Stream logs through TCP/UDP
+Set `type` to **tcp** or **udp** depending of your protocol then specify the `port` of your incomming connection.
+
+Example: 
```

Comment:
If we could avoid to use java which is a supported integration (even if we encourage to log in JSON, still need to find a place to do so btw):

I guess we could use a php example ?
init_config:
instances:
  - whatever: anything
#Log section
logs:
  - type: tcp
    path: 10518
    service: webapp
    source: php
    sourcecategory: front

Reply:
👍

Reply:
@l0k0ms  Github removes the `-`and replace them by points, we however need those dash in the examples

Reply:
👍